src/locations/service/LocationsService.ts

```diff
@@ -44,7 +44,17 @@ export default class LocationsService extends RestService {
     });
     return deferred.promise as any;
   }
-
+  public updateSla(locationId: string, sla: LocationSLA): Promise<LocationSLA> {
+    const deferred = Q.defer<Location>();
+    const requestUrl = `/${Version.V1}${Resource.LOCATIONS}/${locationId}/sla`;
+    const options = {
+      data: sla
+    };
+    this.client.put(requestUrl, options, (error, response, body) => {
+      return this.mapResponseToPromise(requestUrl, deferred, error, response, body, this.EXPECTED_GET_OR_UPDATE_STATUS);

```

Comment:
Similar to my comment of the closures PR, can you add a method to map the response to the LocationSLA object?

I noticed for the `getSla()` method it doesn't exist either, so please add in both methods and update the tests accordingly.

The only mapping which will be required is for the date fields:

```
LocationSLA
{
  createdAt: Date; // Map from ISO string to date
  fulfillmentSLAHours?: string;
  locationId: string;
  operatingDays: LocationSLAOperatingDay[];
  updatedAt: Date; // Map from ISO string to date
}
```
And be sure to loop through each LocationSLAOperatingDay in the array and do the same:

```
LocationSLAOperatingDay
{
  createdAt: Date; // Map from ISO string to date
  day: string;
  end: string;
  fulfillmentCutoffTime: string;
  id: string;
  open: string;
  updatedAt: Date; // Map from ISO string to date
}
```